Tools/autotest/param_metadata/param_parse.py

```diff
@@ -335,5 +335,10 @@ def do_emit(emit):
     do_emit(HtmlEmit())
     do_emit(RSTEmit())
     do_emit(MDEmit())
+    try:
+        from ednemit import EDNEmit
+        do_emit(EDNEmit())
+    except:
+        print("Unable to emit EDN, install edn_format if desired")
```

Comment:
Exception would have to be tighter to justify this error message.

Reply:
Tightened to just `ImportError` is that fine?